views/game.hbs

```diff
@@ -35,7 +43,8 @@
           </label>
         </form>
         {{else}}
-        <div class="cell" data-color="{{this.value}}">
+        <div class="cell" data-color="{{this.value}}"
+          data-winningCell="{{isCellInWinningPath @root.state.winningPath @index @../index}}">
```

Comment:
I find the syntas `@../index` cryptic. Can't you name the index variable in the `#each` block?